chart/env/prod.yaml

```diff
@@ -181,7 +180,7 @@ mongodbMigration:
 
 cacheMaintenance:
   action: "skip"
-  # ^ allowed values are {backfill, collect-queue-metrics, collect-cache-metrics, delete-indexes, post-messages, skip}
+  # ^ allowed values are {backfill, collect-queue-metrics, collect-cache-metrics, clean-duckdb-index-downloads, clean-duckdb-index-job-runner, post-messages, skip}
```

Comment:
I think clean-hf-datasets-cache is missing.

It's a lot of different actions for the same thing. Maybe use only one `clean-directory` action, and differentiate using environment variables? or is it too risky?